backend/entityservice/database/selections.py

```diff
@@ -387,7 +394,19 @@ def get_all_objects_for_project(db, project_id):
               runs.run_id = similarity_scores.run AND
               runs.project = %s
             """, [project_id])
-        similarity_files = [res['file'] for res in query_response]
-        object_store_files.extend(similarity_files)
+    similarity_files = [res['file'] for res in query_response]
+    return similarity_files
 
-    return object_store_files
+
+def get_similarity_file_for_run(db, run_id):
+    query_response = query_db(db, """
+            SELECT 
+              similarity_scores.file
+            FROM 
+              similarity_scores
+            WHERE 
+              similarity_scores.run = %s
+            """, [run_id])
+    similarity_files = [res['file'] for res in query_response]
+    if len(similarity_files):
+        return similarity_files[0]
```

Comment:
Should we log something if there is more than 1 file? I know it should not happen, but what if?